chromadb/api/types.py

```diff
@@ -292,6 +296,10 @@ def validate_ids(ids: IDs) -> IDs:
     for id_ in ids:
         if not isinstance(id_, str):
             raise ValueError(f"Expected ID to be a str, got {id_}")
+
+        if len(id_) == 0:
```

Comment:
On the fence about this still.

Reply:
we can discuss offline

Reply:
The empty string is a valid identifier, and would still be unique - I have no idea why anyone would want to do this, but let's let them.